playwright/pages/components/InAppGuideDialog.ts

```diff
@@ -0,0 +1,38 @@
+import { Locator, Page } from "@playwright/test";
+
+import { Dialog } from "./Dialog";
+
+export class InAppGuideDialog extends Dialog {
+  override readonly closeButton: Locator;
```

Comment:
I'm not yet 100% sure how to correctly target editor-ui close button. We may need a small update on the editor part.

Reply:
Indeed, I had to add a custom data property to the close buttons for the Page Builder demo (https://prismic.io/try). Doing more complex selectors seems fragile (e.g. `div > div > header > button`).

Reply:
Yes, We don't want to do complex selectors like this. Better to add a test id or an aria something in the editor package.